ghcide/src/Development/IDE/Core/Compile.hs

```diff
@@ -996,7 +996,10 @@ initTypecheckEnv hsc_env mod = initTc hsc_env HsSrcFile False mod fakeSpan
     fakeSpan :: RealSrcSpan
     fakeSpan = realSrcLocSpan $ mkRealSrcLoc (Util.fsLit "<ghcide>") 1 1
 
-
+-- | Non-interactive handling of the module interface.
+--   A non-interactive modification of code from the 'GHC.Runtime.Eval.getDocs'.
+--   The interactive paths create problems in ghc-lib builds
+---  and lead to fun errors like "Cannot continue after interface file error".
```

Comment:
Now the most informative doc belongs here, where the noninteractive interface abstraction is.